modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_empty_drop_zone.scss

```diff
@@ -1,40 +1,34 @@
 .empty-drop-zone-root {
+	background: $secondaryHover;
+	border-width: 2px;
+
+	.drop-zone-target {
+		height: 86px;
+	}
+
 	.empty-drop-zone-target {
+		height: 462px;
```

Comment:
Height in pixels can be a pain in the future. Lets try to use a % or a viewport related unit.